cli/train.go

```diff
@@ -68,6 +94,31 @@ func (c *viamClient) dataSubmitTrainingJob(datasetID, orgID, modelName, modelVer
 	return resp.Id, nil
 }
 
+// dataSubmitTrainingJob trains on data with the specified filter.
+func (c *viamClient) dataSubmitCustomTrainingJob(datasetID, registryItemID, orgID, modelName,
```

Comment:
I think we can change this to mlSubmitCustomTrainingJob and then the description would be to train on data with a specified dataset and registry item

Reply:
Gah! 🥚 🫤 

For those at home trying to follow along -- this means egg on face.